api/service/inst-api-functions.sh

```diff
@@ -45,7 +39,7 @@ backup_old_api() {
 
     # check current REVISION and perform the applicable backup
     if [ "$1" -ge 40000 ]; then
-        stop_api_4x
+        # API doesn't need to be stopped individually in Wazuh 4.0 or later as it is integrated as a daemon.
```

Comment:
This comment is only necessary if you leave commented `stop_api_4x`. Otherwise, a user can read the comment but he can't imagine that there was a stop_api function call.

Reply:
Removed